docs/index.html

```diff
@@ -35,390 +36,292 @@
+            {name: "MAINTAIN", color: "#efafa9"}
         ],
         print_layout: true,
         // zoomed_quadrant: 0,
         //ENTRIES
         entries: [
             {
-                "quadrant": LANGUAGE,
-                "ring": TRIAL,
+                "quadrant": DEVOPS,
+                "ring": CORE,
                 "label": "Jenkins",
                 "active": true,
-                "moved": 0
-            },
-            {
-                "quadrant": LANGUAGE,
-                "ring": ADOPT,
-                "label": "Percy",
-                "active": true,
-                "moved": 0
-            },
-            {
-                "quadrant": LANGUAGE,
-                "ring": ADOPT,
-                "label": "Scrutinizer",
-                "active": true,
-                "moved": 0
-            },
-            {
-                "quadrant": LANGUAGE,
-                "ring": ADOPT,
-                "label": "SonarQube",
-                "active": true,
-                "moved": 0
-            },
-            {
-                "quadrant": LANGUAGE,
-                "ring": ADOPT,
-                "label": "Lighthouse",
-                "active": true,
-                "moved": 0
-            },
-            {
-                "quadrant": LANGUAGE,
-                "ring": ADOPT,
-                "label": "Cypress",
-                "active": true,
-                "moved": 0
-            },
-            {
-                "quadrant": LANGUAGE,
-                "ring": ADOPT,
-                "label": "Github",
-                "active": true,
-                "moved": 0
-            },
-            {
-                "quadrant": LANGUAGE,
-                "ring": TRIAL,
-                "label": "Gitlab",
-                "active": true,
-                "moved": 0
+                "moved": 1
             },
-            {
-                "quadrant": LANGUAGE,
-                "ring": ADOPT,
+            // {
+            //     "quadrant": LANGUAGE,
+            //     "ring": COMPETENT,
+            //     "label": "Percy",
+            //     "active": true,
+            //     "moved": -1
+            // },
+            // {
+            //     "quadrant": LANGUAGE,
+            //     "ring": COMPETENT,
+            //     "label": "Scrutinizer",
+            //     "active": true,
+            //     "moved": -1
+            // },
+            // {
+            //     "quadrant": LANGUAGE,
+            //     "ring": COMPETENT,
+            //     "label": "SonarQube",
+            //     "active": true,
+            //     "moved": -1
+            // },
+            // {
+            //     "quadrant": LANGUAGE,
+            //     "ring": COMPETENT,
+            //     "label": "Lighthouse",
+            //     "active": true,
+            //     "moved": -1
+            // },
+            // {
+            //     "quadrant": LANGUAGE,
+            //     "ring": COMPETENT,
+            //     "label": "Cypress",
```

Comment:
Shouldn't we add the core tools for frontend like cypress and percy?

Reply:
That is CICD right? I'm doubting if it makes sense to include CICD tooling, what does everybody think about that?

Reply:
CICD tooling that we like to use is not all that important I think, but then the same could be said for Bitrise and Github Actions. It also depends on wether this Tech Radar is more for internal use (which tools do we typically use to get the job done) or the outside world (what tools does Harborn have experience with that can help my company).